src/app/modules/shared-components/components/input-error/input-error.component.html

```diff
@@ -0,0 +1,5 @@
+<div *ngIf="control && control.errors" style="color: #a94442">
```

Comment:
`control && control.errors` can be `control?.errors`

Reply:
This doesn't seem do be working